modules/raft/src/main/java/org/apache/ignite/internal/raft/storage/impl/RocksDbSharedLogStorage.java

```diff
@@ -170,8 +169,7 @@ public class RocksDbSharedLogStorage implements LogStorage, Describer {
         this.groupEndPrefix = groupEndPrefix(groupId);
         this.groupStartBound = new Slice(groupStartPrefix);
         this.groupEndBound = new Slice(groupEndPrefix);
-
-        this.writeOptions = new WriteOptions();
+        this.writeOptions = writeOptions;
```

Comment:
`writeOptions` should not be closed by this instance as it's not managed by it anymore